src/CraftingRecipes.h

```diff
@@ -143,7 +143,7 @@ class cCraftingRecipes
 	cRecipe * getRecipeById(UInt32 a_RecipeId);
 
 	/** Gets a map of all recipes with name and recipe id */
-	std::map<AString, UInt32> getRecipeNameMap();
+	std::map<AString, UInt32> & getRecipeNameMap();
```

Comment:
```suggestion
	const std::map<AString, UInt32> & getRecipeNameMap();
```

Reply:
You are a bit too fast for me, I'm currently fixing my build environment, so that I can test it properly. I only commited and pushed so that I'm able to check it out cleanly, I'll let you know once I'm satisfied :-)

(While I wouldn't have spotted this (and the following) suggestions, so thanks)